File: 151004/Tishalovich/rest_basics/publisher/src/main/java/by/bsuir/dc/publisher/controllers/StoryController.java
package by.bsuir.dc.publisher.controllers;

import by.bsuir.dc.publisher.entities.dtos.request.StoryRequestTo;
import by.bsuir.dc.publisher.entities.dtos.response.StoryResponseTo;
import by.bsuir.dc.publisher.services.common.AbstractService;
import by.bsuir.dc.publisher.controllers.common.AbstractController;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("api/v1.0/storys")
public class StoryController extends AbstractController<StoryRequestTo, StoryResponseTo> {

    public StoryController(AbstractService<StoryRequestTo, StoryResponseTo> service) {
        super(service);
    }

}

File: 151004/Akimenko/DC_task_05/redis/Publisher/Controllers/V1_0/StoryController.cs
using AutoMapper;
using Microsoft.AspNetCore.Mvc;
using Publisher.Controllers.V1_0.Common;
using Publisher.Entity.Db;
using Publisher.Entity.DTO.RequestTO;
using Publisher.Entity.DTO.ResponseTO;
using Publisher.Service.Interface;

namespace Publisher.Controllers.V1_0
{
    [Route("api/v1.0/storys")]
    [ApiController]
    public class storyController(IstoryService storyService, ILogger<storyController> Logger, IMapper Mapper)
        : AbstractController<story, storyRequestTO, storyResponseTO>(storyService, Logger, Mapper)
    { }
}
